zest/releaser/release.py

```diff
@@ -98,7 +98,7 @@ def _make_tag(self):
         for cmd in cmds:
             print(cmd)
             if utils.ask("Run this command"):
-                print(execute_command(cmd))
+                logger.info(execute_command(cmd))
```

Comment:
Doing this as a log message breaks the "layout" of the printed information a bit.

Any idea why logger.info() would work fine and print() not? Can you provide some context for us?